transforms/paths/schema.js

```diff
@@ -1,9 +1,10 @@
 const errorMessage = require('../utils/errorMessage');
 const combineSchema = require('../utils/combineSchema');
+const addDefaultValue = require('../utils/defaultValue');
 const addEnumValues = require('../utils/enumValues');
 const { refSchema, formatRefSchema } = require('../utils/refSchema');
 
-const getSchema = (entity, message) => (type, enumValues = [], jsonOptions = {}) => {
+const getSchema = (entity, message) => (type, defaultValue = '', enumValues = [], jsonOptions = {}) => {
```

Comment:
I don't think we should add a default value for this property: it should be optional and, therefore, remain as `undefined` if not specified. 

Also, depending on the type, an empty string wouldn't always make sense. On a numeric field, for example, we may want need to use a zero or something else, as an empty string could be considered a `NaN` value. For boolean fields, a default of `true` or `false` would make more sense. These are just examples, but this is to illustrate that we should not use an empty string as a default for every field.